index.js

```diff
@@ -551,6 +542,13 @@ function fixNonStringValue(value) {
     return value;
   }
 
+  if (value === 'True') {
```

Comment:
can value be `"true"` or `"TRUE"`? Maybe downcase the value and then compare to `true`?